### ISSUE TITLE
Is there a way for client to ask relay to probe for bandwidth 

### ISSUE THREAD

**Issue by fluffy** [moq-wg/moq-transport#370]
This keeps coming up so wanted to keep a place holder here.

**Comment by ianswett** [comment]
No, but I'd like to resolve a number of other issues first, such how ABR is going to work.  My past experience is that this is extremely difficult to get right, so doing so for a general purpose application is extraordinarily difficult.

**Comment by ianswett** [comment]
This is a more general version of #534.

In today's conversation, there was interest in proactively retransmitting high priority data to probe for bandwidth, but it's not clear what layer that should be at?  Doing it at the QUIC layer is nice because if either STREAM frame is ACKed, it won't get sent again, but I don't see WebTransport exposing an API like that.

This is an area for more thought about what the best approaches might be.

**Comment by jlivingood** [comment]
Is SCONE useful here? 
https://www.ietf.org/archive/id/draft-ietf-scone-protocol-03.html

**Comment by fluffy** [comment]
Given this is more one of the key uses cases for SCONE, I think we should talk about how to use SCONE info here if it is available

**Comment by ianswett** [comment]
The WG was largely supportive of closing this with no action, though we could add some editorial text on SCONE.

**Comment by acbegen** [comment]
Previously, we had implemented this in a different way. We created a probing track (sending periodic probes) and whenever the subscriber needed a probing test, it would subscribe to this track and then unsubscribe. There are subtleties regarding whether the probing traffic should be prioritized or not.

**Comment by ianswett** [comment]
Yes, using a probing track is definitely an option that MoQT would support.
